pkgs/cosmic-session/package.nix

```diff
@@ -51,6 +51,9 @@ rustPlatform.buildRustPackage {
     "--set"
     "prefix"
     (placeholder "out")
+    "--set"
+    "cosmic_dconf_profile"
+    "/etc/dconf/profile/cosmic"
```

Comment:
I think just `cosmic` and letting dconf use its normal method to find profiles again would be slightly better (and more portable for using this package on nix-y but not exactly NixOS operating systems). But also I don't feel that strongly :)

Reply:
good call, I made a mental note of that when reading the PR and completely forgot haha